api/src/main/java/org/pmiops/workbench/monitoring/attachments/AttachmentKey.java

```diff
@@ -0,0 +1,23 @@
+package org.pmiops.workbench.monitoring.attachments;
+
+public enum AttachmentKey implements AttachmentKeyBase {
```

Comment:
Why did you separate out the AttachmentKeyBase interface? This pattern isn't used elsewhere in our codebase AFAICT, and it's non-obvious to me why a trivial interface was pulled out. Worth a comment somewhere at least.

Reply:
New version supports allowed Metrics and Discrete Values, enforced by the MeasurmentBundle builder.